changelogs/unreleased/ghe-523-max-config-param-length.md

```diff
@@ -1,3 +1,3 @@
-## feature/box
+## bugfix/box
 
-* The maximum length of `box.cfg{}` string parameters is now 512 instead of 256.
```

Comment:
It looks almost OK. I can understand that there was a problem with some valid configuration due to the old limit. However, it would be better to say it explicitly.

Reply:
fixed